src/test/groovy/edu/ie3/datamodel/models/LoadProfileTest.groovy

```diff
@@ -0,0 +1,121 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+package edu.ie3.datamodel.models
+
+import edu.ie3.datamodel.models.profile.BdewStandardLoadProfile
+import edu.ie3.datamodel.models.profile.LoadProfile
+import edu.ie3.datamodel.models.profile.NbwTemperatureDependantLoadProfile
+import spock.lang.Specification
+
+class LoadProfileTest extends Specification {
+	def "Load profiles are parsed correctly from correct input" () {
```

Comment:
I also would've loved to see a test for throwing the exception when providing an unknown key, but this was not implemented before, so I wouldn't expect you to implement it.  Just if you feel like it...